mesonbuild/backend/ninjabackend.py

```diff
@@ -1843,9 +1843,6 @@ def scan_fortran_module_outputs(self, target):
                     modmatch = modre.match(line)
                     if modmatch is not None:
                         modname = modmatch.group(1).lower()
-                        if modname == 'procedure':
```

Comment:
Why is this no longer possible?

Reply:
It simply is not possible for this token to appear with the above more specific regex.
in the future, more advanced parsing would use a set of regex matches and nested `if elif`, as the set of possibilities following `module` has several possibilities--even CMake can't handle all of them.

Reply:
Ah yes, I missed the `$` in the regex.